src/mux/comment.ts

```diff
@@ -99,22 +99,11 @@ Mux.post<IComment>(
       const result = await imComment.save();
       const savedComment = await ModelComment.findById(result._id).populate(['author', 'article']);
       if (savedComment) {
-        const { uuid, author, article, reply, votedUser, content, created, updated, hidden, vote } = <IComment>(
-          savedComment.toObject()
-        );
+        const responseComment = <IComment>savedComment.toObject();
```

Comment:
We don't want to expose unnecessary information that's why we only pick several fields to return. Please aware of `_id` could be exposed.